### ISSUE TITLE
It's not embrace-extend-extinguish, it's embrace-extend-monopolize-monetize

### ISSUE THREAD

**Issue by qwertychouskie** [upend/IF_MS_BUYS_GITHUB_IMMA_OUT#80]
People keep saying "embrace-extend-extinguish" in regards to the buyout, but I believe that is not right.  Why would they extinguish a platform they just payed $7.5billion for when they have no directly competing platform?  Look at e.g. Minecraft:

- Embrace:  MS buys Mojang
- Extend:  The "Better Together Update", which lets e.g. console players play with Windows 10 edition players (see https://www.pcgamesn.com/minecraft/microsoft-minecraft-java-edition)
- Monopolize:  Sorry, the Java edition doesn't get the update, use the Windows 10 edition instead.
- Monetize:  e.g. the "Minecraft Marketplace" (https://techcrunch.com/2017/04/10/minecraft-is-getting-a-marketplace-where-creators-can-monetize-their-creations/)

It's embrace-extend-monopolize-monetize.  It's just a matter of time before it happens to Github.

P.S.  Maybe an edited version of this should be added to the readme?

**Comment by vassudanagunta** [comment]
i agree with your assessment. but let’s focus on providing options for people who want to leave. living our convictions is the best way to promote change.
